docs/getting-started.md

```diff
@@ -62,8 +67,63 @@ Under most circumstances you won't need to run the application in Docker locally
 - copy the `.env.example` file, save as `.env` and populate the secrets within
 - run `docker-compose up`
 
+## Run tests
+
+### Unit tests
+
+To run the project's unit tests, open a terminal in the project directory and run:
```

Comment:
Also add about using your preferred IDE - not many people will run tests from command line